doc/api/esm.md

```diff
@@ -117,16 +117,19 @@ The resolve hook returns the resolved file URL and module format for a
 given module specifier and parent file URL:
 
 ```js
-import url from 'url';
+const baseURL = new URL('file://');
+baseURL.pathname = process.cwd() + '/';
 
 export async function resolve(specifier, parentModuleURL, defaultResolver) {
   return {
-    url: new URL(specifier, parentModuleURL).href,
+    url: new URL(specifier, parentModuleURL || baseURL).href,
```

Comment:
Given we use the pattern `parentModuleURL = baseURL` below for the default argument value - it's probably a good idea to stay consistent in the docs here and use it too.